src/umpire/strategy/QuickPool.cpp

```diff
@@ -312,11 +312,11 @@ void QuickPool::coalesce() noexcept
 {
   UMPIRE_LOG(Debug, "()");
 
-  umpire::event::record([&](auto& event) {
-    event.name("coalesce").category(event::category::operation).tag("allocator_name", getName()).tag("replay", "true");
```

Comment:
This is needed by the `replay` tool and should not be removed.

Reply:
Thanks for letting me know! I'll remove it

Reply:
It looks like the (failing) tests are letting you know as well.  I believe if you put these lines back, the replay tests will succeed again.